rmgpy/data/thermo.py

```diff
@@ -1731,8 +1736,9 @@ def getBicyclicCorrectionThermoDataFromHeuristic(self, bicyclic):
                 aromaticBonds = findAromaticBondsFromSubMolecule(submol)
                 for aromaticBond in aromaticBonds:
                     aromaticBond.setOrderNum(1)
-                
-                submol.update()
+
+                submol.saturate_unfilled_valence()
+                submol.update
```

Comment:
Doesn't `saturate_unfilled_valence` already update the molecule?

Reply:
Yeah you're correct, I will take out those lines.